chapter05/jxp/ch5ex1.pl

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/perl
+use warnings;
+use strict;
+
+my @lines;
+
+while( my $line = <> ){
+  if ( $line !~ /\n$/  ) {
+    $line .= "\n";
```

Comment:
It's in case one of the files doesn't end in a new line. Then you are missing a newline in the middle of the loop